python/ray/train/v2/_internal/callbacks/accelerators.py

```diff
@@ -63,7 +64,7 @@ def _share_cuda_visible_devices(worker_group: WorkerGroup):
         - Worker2: "0,1"
     """
     _share_accelerator_ids(
-        worker_group, ray_constants.GPU, ray_constants.CUDA_VISIBLE_DEVICES_ENV_VAR
+        worker_group, ray_constants.GPU, CUDA_VISIBLE_DEVICES_ENV_VAR
```

Comment:
@justinvyu PTAL for codeowner approval, and also can you help me understand why we depend on this env var directly?